### ISSUE TITLE
Is this project deprecated?

### ISSUE THREAD

**Issue by vinyll** [ChuckJonas/ts-force#141]
I am about to write an app based on this project.
It should be resilient.

is this ts-force lib usable, maintained and ok for production?

**Comment by ChuckJonas** [comment]
I'm not actively adding new features, but the library is production stable and I will continue with patches and welcome community support (I'm working on closing the open PR's just have been extremely busy lately).

**Comment by vinyll** [comment]
Right, it makes sense.
It seems to be setup for version 50 by default. Any experience with more recent versions (v58)?

I notice your project was [forked and continued by @vacasaoss](https://github.com/vacasaoss/ts-force/), however I havent checked if the work done was specific or generic enough to be considered.

**Comment by ChuckJonas** [comment]
@vinyll Salesforce is pretty good about introducing breaking changes to the foundational API contract so really I wouldn't expect there to be any issues.  

I just updated locally to v58 and ran all the tests (which are actually fairly comprehensive) and everything passes as expected.  I'll probably go ahead and publish with default version on `v58`.

Now that I just released @colinfindlay-nz changes, the differences [don't seem to be very substantial between the two](https://github.com/ChuckJonas/ts-force/compare/master...vacasaoss:ts-force:master).  Guessing, they just got tied of waiting on me to release the updates. 

Maybe @colinfindlay-nz could speak to his experience with ts-force in terms of "production" readiness
